README.md

```diff
@@ -33,60 +33,53 @@ developers, use the smaller
 
 ## Deploy the bundle
 
-```
 
+```
 juju deploy canonical-kubernetes
-
 ```
 
-This will deploy the Canonical Kubernetes offering with default constraints.
-This is useful for lab environments, however for real-world use you should
-provide high CPU and memory instances to kubernetes-worker.
+This will deploy the Canonical Distribution of Kubernetes offering with default
+constraints. This is useful for lab environments, however for real-world use
+you should provide higher CPU and memory instances to kubernetes-worker units.
 
-You can do this by editing the [bundle](https://github.com/juju-solutions/bundle-canonical-kubernetes)
-to fit your needs, it is commented for your convenience.
+You can increase the constraints by editing the
+[bundle.yaml](https://github.com/juju-solutions/bundle-canonical-kubernetes)
+to fit your needs by removing the `#` comment character.
 
 ```
-
 juju deploy ./bundle.yaml
-
 ```
 
-This bundle exposes the kubeapi-load-balancer and kibana applications by default.
-This means those charms are accessible through the public addresses.
+This bundle exposes the kubeapi-load-balancer and kibana applications by 
+default. This means those charms are accessible through the public addresses.
 
 If you would like to remove external access, unexpose the applications:
 
 ```
-
 juju unexpose kibana
 juju unexpose kubernetes
```

Comment:
There is no 'kubernetes' application now, its either kubernetes-master or kubernetes-worker